app/views/pages/_news.haml

```diff
@@ -8,33 +8,14 @@
     .row
       .col-md-12.news-item
         %ul.timeline
-          %li.timeline-inverted
-            .timeline-image
-              %a{ :target => "_blank", :href => "http://www.meetup.com/fr-FR/parisrb/" }
-                %img.img-circle.img-responsive{ :alt => "ParisRb", :src => image_path("news/parisrb.png") }/
-            .timeline-panel
-              .timeline-heading
-                = t('parisrb_2_date')
-                .subheading Paris.rb
-              .timeline-body
-                %p= t('parisrb_2_desc_html')
-          %li
-            .timeline-image
-              %a{ :target => "_blank", :href => "https://www.facebook.com/events/1678846872385676/" }
-                %img.img-circle.img-responsive{ :alt => "Hopdrinks", :src => image_path("news/hopdrink-janvier.png") }/
-            .timeline-panel
-              .timeline-heading
-                = t('hopdrinks_date')
-                .subheading Hopdrinks
-              .timeline-body
-                %p= t('hopdrinks_desc_html')
-          %li.timeline-inverted
-            .timeline-image
-              %a{ :target => "_blank", :href => "http://www.meetup.com/fr-FR/parisrb/" }
-                %img.img-circle.img-responsive{ :alt => "ParisRb", :src => image_path("news/parisrb.png") }/
-            .timeline-panel
-              .timeline-heading
-                = t('parisrb_1_date')
-                .subheading Paris.rb
-              .timeline-body
-                %p= t('parisrb_1_desc_html')
+          -NEWS.reverse_each.with_index do |news,i|
+            %li{class: ("timeline-inverted" if i%2==0)}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace